rbac/management/role/definitions/approval.json

```diff
@@ -0,0 +1,140 @@
+{
+  "roles": [
+    {
+      "name": "Approval Administrator",
+      "system": true,
+      "version": 1,
```

Comment:
Need to make this version `2` til I get a change in to add this role to existing accounts.

Reply:
Changed